README.md

```diff
@@ -155,6 +57,12 @@ Use "picotool help <cmd>" for more info
 
 Note commands that aren't acting on files require a device in BOOTSEL mode to be connected.
 
+## Building & Installing
+
+If you don't want to build picotool yourself, you can find pre-built executables for Windows, macOS, and Linux in the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository. Assuming you've unzipped that directory to `$PICOTOOL_LOC` (with the actual picotool executable at `$PICOTOOL_LOC/picotool/picotool`), you can point the Pico SDK at this binary by setting the `picotool_DIR` environment variable to `$PICOTOOL_LOC/picotool`, or by passing `-Dpicotool_DIR=$PICOTOOL_LOC/picotool` to your `cmake` command or setting it in your `CMakeLists.txt` file.
```

Comment:
"unzipped that directory" doesn't quite sound right?

Reply:
Yeah, I wasn't really sure how to phrase it - I want to say that if you download it to Downloads/xxx.zip, extract that to Downloads/xxx, then the picotool binary is at Downloads/xxx/picotool/picotool - the crucial point is that it's not at Downloads/xxx/picotool

Reply:
I think that depending on the "unzip" program being used and/or which unzip options are chosen, you might end up with `picotool-2.1.1-x64-win\picotool\picotool.exe` or you might end up with just `picotool\picotool.exe` ?
Either way, I think "extracted that archive" might sound better than "unzipped that directory"?